browser/mapping.go

```diff
@@ -253,6 +253,9 @@ func mapElementHandle(vu moduleVU, eh *common.ElementHandle) mapping {
 		if err != nil {
 			return nil, err //nolint:wrapcheck
 		}
+		if eh == nil {
```

Comment:
nit: We might want to add a comment here (and in the other cases) similar to: _"ElementHandle can be nil if selector does not match any elements"_.

Reply:
Sounds good, done in a2dc2585edf7b3ab11254faafd82b50f6e563af9